runtime/compiler/optimizer/VectorAPIExpansion.cpp

```diff
@@ -793,19 +795,6 @@ TR_VectorAPIExpansion::expandVectorAPI()
          TR::Node::recreate(parent, TR::treetop);
          methodTable[handlerIndex]._methodHandler(this, treeTop, node, elementType, vectorLength, doMode);
          }
-      else if (node->getOpCodeValue() == TR::checkcast)
```

Comment:
I take it that you agree that this code was dead :sweat_smile: 

The `checkcast`-related deletion(s) seem to be unrelated to the other changes in this commit though, so I think they warrant a separate commit or at least a separate mention in the commit message

Reply:
yeah, added to the commit message.